spec/service_consumers/pact_helper.rb

```diff
@@ -79,6 +82,13 @@ def url_encode(str)
     allow(UserAttributes).to receive(:load_config_file).and_return(fixture_file)
 
     stub_request(:post, Plek.find("account-manager") + "/api/v1/jwt").to_return(status: 200, body: { id: "jwt-id" }.to_json)
+
+    %w[/guidance/some-govuk-guidance].each do |page_path|
+      stub_content_store_has_item(
+        page_path,
+        content_item_for_base_path(page_path).merge("content_id" => SecureRandom.uuid),
+      )
+    end
```

Comment:
Why use an array here if we only have one item? Is this the same as
```ruby
page_path = "/guidance/some-govuk-guidance"

stub_content_store_has_item(
  page_path,
  content_item_for_base_path(page_path).merge("content_id" => SecureRandom.uuid),
)
```
?

Reply:
Yeah, the array isn't needed.  I'll change it.